regression_em/model.py

```diff
@@ -1,18 +1,287 @@
+"""Package to perform regression EM algorithm self.
+For the detail of this algortithm,
+["Position Bias Estimation for Unbiased Learning to Rank in Personal Search" Xuanhui Wang et al.]
+(https://static.googleusercontent.com/media/research.google.com/ja//pubs/archive/46485.pdf)
+"""
+from random import random
+# from typing import Sequence, Tuple, Union
+from typing import Tuple, Union
+
+import numpy as np
 import scipy.sparse as sp
-from sklearn.linear_model import LinearRegression
+from sklearn.linear_model import LinearRegression, Ridge
+from sklearn.base import BaseEstimator
+
+
+# Define type.
+Matrix = Union[np.ndarray, sp.csr_matrix, sp.csc_matrix]
+RegParam = Tuple[Matrix, float]
```

Comment:
A coefficient vector of linear regression is numpy array.

Reply:
The definition of RegParam is fixed.
`RegParam = Tuple[np.array, float]`